charts/bpndiscovery/values.yaml

```diff
@@ -47,6 +47,7 @@ bpndiscovery:
       clientId: default-discovery-finder-client
       clientSecret: ""
       authorizationGrantType: client_credentials
+    schedulerCronFrequency: ""
```

Comment:
Would the application starts with the default values? 

https://github.com/eclipse-tractusx/sldt-bpn-discovery/blob/b70aa33a5e943ed5efb0fc693247cb740b024b36/backend/src/main/java/org/eclipse/tractusx/bpndiscovery/service/RegisterService.java#L49